build-windows.bat

```diff
@@ -0,0 +1,4 @@
+cmake -DCMAKE_TOOLCHAIN_FILE=C:\work\vcpkg\scripts\buildsystems\vcpkg.cmake .
```

Comment:
I tried this in windows.  can't find pthreads.h.

Reply:
This works once I build using VS 2017 command prompt.  As a side note, I wonder if this is a good place to document windows build dependencies.

---
In reply to: [192441211](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192441211) [](ancestors = 192441211)